install.sh

```diff
@@ -240,23 +240,19 @@ nvm_check_global_modules() {
   )"
 
   if [ "${MODULE_COUNT}" != '0' ]; then
-    cat <<-'END_MESSAGE'
-	=> You currently have modules installed globally with `npm`. These will no
-	=> longer be linked to the active version of Node when you install a new node
-	=> with `nvm`; and they may (depending on how you construct your `$PATH`)
-	=> override the binaries of modules installed with `nvm`:
+    echo "=> You currently have modules installed globally with `npm`. These will no"
+    echo "=> longer be linked to the active version of Node when you install a new node"
+    echo "=> with `nvm`; and they may (depending on how you construct your `$PATH`)"
```

Comment:
whoops, looks like shellcheck is still failing. Let's single-quote all these lines since we're not doing any interpolation.